SECURITY.md

```diff
@@ -1,6 +1,6 @@
 # Reporting Security Issues
 
-Please report any security issues you discover to security[at]citizenlab[dot]co
+At Go Vocal (formerly CitizenLab) we take security seriously. Please report any security issues you discover to security[at]govocal[dot]com
```

Comment:
Added the `formerly CitizenLab` part so that it's obvious why the Github URL is still a citizenlab one